homeassistant/components/isy994/strings.json

```
@@ -6,7 +6,7 @@
       "user": {
         "data": {
           "username": "[%key:common::config_flow::data::username%]",
-          "host": "URL",
+          "host": "[%key:common::config_flow::data::url%]",
```

Should it be : 
```suggestion
          "host": "[%key:common::config_flow::data::host%]",
```
?
Hard to say, I don't know the integration at all but looks weird to have host => URL no ?

---

It does look weird. I'll wait for one of the codeowners or someone else familiar with the code to make the final decision.

---

The parameter is expecting the full connection URL to the ISY host in the format `http(s)://<ip, hostname, or my.isy.io>:<port>` and an optional unique root path at the end if connecting to the ISY portal (similar to Nabu Casa).

`host` is consistent with the documentation, but I'm not heartset on either.

---

If an URL should be set here, better to use URL key 👍 
Means you were right with the URL key, but you **should** also change the key name to `url` and update the `config_flow.py` and tests.

---

Be careful if the key is used in the config entry data. Changing it then doesn't work easily.

I'd stay away from changing input form key names. Abort and error keys are safe to change since they will never be part of the config entry data.

---

Ok so then let this key as is. Changing the key value is still safe even for data inputs right ?

---

Yes, the values in the strings.json file are only used for translation.

---

If I understand correctly, leave PR as is?